common/menu-entry.c

```diff
@@ -293,6 +294,41 @@ bool menuEntryLoad(menuEntry_s* me, const char* name, bool shortcut) {
         /*if (shortcut)
             shortcutFree(&sc);*/
     }
+    if (me->type == ENTRY_TYPE_THEME) {
+        config_t cfg = {0};
+        config_init(&cfg);
+        config_setting_t *themeInfo;
+        const char *name, *author, *version;
+        bool noAuth = false, noVers = false;
```

Comment:
Get rid of these and just initialize the default string values here.

Reply:
done